src/components/modals/ReminderModal.vue

```diff
@@ -82,6 +82,31 @@ export default {
           }))
         ];
       });
+
+      // add out of script traveller reminders
+      this.$store.state.extraTravellers.forEach(role => {
+        if (players.some(p => p.role.id === role.id)) {
+          reminders = [
+            ...reminders,
+            ...role.reminders.map(name => ({
+              role: role.id,
+              image: role.image,
+              name
+            }))
+          ];
+        }
+        if (role.remindersGlobal && role.remindersGlobal.length) {
```

Comment:
I'm not a fan of this. There's currently no traveler that has a `globalReminder` and if there ever was, that would mean that their `globalReminder` would show up for every and all games (except homebrew), because it would be either in the regular `roles` list or in the `extraTravellers` list, from where it would get added to the list of available reminders.
Travelers should (by definition) not have `globalReminders`, and even if they did, we shouldn't show them unless they're actually part of the script.